tests/testthat/data/sample_config.json

```diff
@@ -0,0 +1,58 @@
+{
+    "job_id": "6183da58-89bc-455f-8562-4f607257a876",
+    "task_id": "bc0c3eb3-7158-4631-a2a9-86b97357f97e",
+    "as_of_date": "2023-10-28",
+    "disease": "test",
+    "geo_value": "test",
+    "geo_type": "test",
+    "min_reference_date": "2023-01-02",
+    "max_reference_date": "2023-01-07",
+    "report_date": "2023-10-28",
+    "quantile_width": [0.5, 0.95],
+    "model": "EpiNow2_test",
+    "parameters": {
+       "generation_interval": {
+         "path": "data/test_parameters.parquet",
+         "blob_storage_container": null
+       },
+       "delay_interval": {
+         "path": null,
+         "blob_storage_container": null
+       },
+       "right_truncation": {
+         "path": null,
+         "blob_storage_container": null
+       }
+    },
+    "data": {
+        "path": "data/test_data.parquet",
+        "blob_storage_container": null,
+        "report_date": [
+            "2023-01-01"
+        ],
+        "reference_date": [
+        "2023-01-01",
+        "2022-12-30",
+        "2022-12-29"
+        ]
+    },
+    "seed": 42,
+    "horizon": 14,
+    "priors": {
+        "rt": {
+            "mean": 1.0,
+            "sd": 0.2
+        },
+        "gp": {
+            "alpha_sd": 0.01
+        }
+    },
+    "sampler_opts": {
+        "cores": 1,
+        "chains": 1,
+        "iter_warmup": 50,
```

Comment:
@zsusswein Looks like I need to add these fields (`iter_warmup` and `iter_sampling`) to the new config generation flow. Are there any others I need to update/add? This is what the schema looks like currently: https://github.com/CDCgov/cfa-config-generator/blob/main/utils/epinow2/constants.py#L1 and https://github.com/CDCgov/cfa-config-generator/blob/main/utils/epinow2/functions.py#L232

Reply:
Is there an easy way to access a generated config?

Reply:
Like if I run the service will it give me a config?

Reply:
Yes: click "Run Workflow" on the right here, and fill in some parameters (or leave blank for defaults) https://github.com/CDCgov/cfa-config-generator/actions/workflows/run-workload.yaml

Then go to the storage container `rt-epinow2-config` in Azure and the generated configs should be there under the timestamped job id. LMK if that doesn't work

Reply:
I am SO EXCITED about this.

Reply:
Here's the diff

![image](https://github.com/user-attachments/assets/41393391-21d0-4dcc-a292-c4e0bf128406)


I think the big ones are that I changed the reference date specification format and moved it up, modified the formatting of the parameter keys, and added a few more fields.

I'm also not sure why `as_of_date` is an int?

Reply:
Ok, sounds good -- I'll update the fields on my end accordingly. I think I had set as_of_date as an integer timestamp, but can make it a date string to be consistent with the others

Reply:
implemented here: https://github.com/CDCgov/cfa-config-generator/pull/26